src/platform_impl/windows/dpi.rs

```diff
@@ -2,41 +2,43 @@
 
 use std::sync::Once;
 
+use windows::Win32::{
+    Foundation::{HWND, S_OK},
+    Graphics::Gdi::{
+        GetDC, GetDeviceCaps, MonitorFromWindow, HMONITOR, LOGPIXELSX, MONITOR_DEFAULTTONEAREST,
+    },
+    UI::{
+        HiDpi::{
+            DPI_AWARENESS_CONTEXT, DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE, MDT_EFFECTIVE_DPI,
+            PROCESS_PER_MONITOR_DPI_AWARE,
+        },
+        WindowsAndMessaging::IsProcessDPIAware,
+    },
+};
+
 use crate::platform_impl::platform::util::{
     ENABLE_NON_CLIENT_DPI_SCALING, GET_DPI_FOR_MONITOR, GET_DPI_FOR_WINDOW, SET_PROCESS_DPI_AWARE,
     SET_PROCESS_DPI_AWARENESS, SET_PROCESS_DPI_AWARENESS_CONTEXT,
 };
-use winapi::{
-    shared::{
-        minwindef::FALSE,
-        windef::{DPI_AWARENESS_CONTEXT, DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE, HMONITOR, HWND},
-        winerror::S_OK,
-    },
-    um::{
-        shellscalingapi::{MDT_EFFECTIVE_DPI, PROCESS_PER_MONITOR_DPI_AWARE},
-        wingdi::{GetDeviceCaps, LOGPIXELSX},
-        winuser::{self, MONITOR_DEFAULTTONEAREST},
-    },
-};
 
-const DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2: DPI_AWARENESS_CONTEXT = -4isize as _;
+const DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2: DPI_AWARENESS_CONTEXT = -4;
 
 pub fn become_dpi_aware() {
     static ENABLE_DPI_AWARENESS: Once = Once::new();
     ENABLE_DPI_AWARENESS.call_once(|| {
         unsafe {
             if let Some(SetProcessDpiAwarenessContext) = *SET_PROCESS_DPI_AWARENESS_CONTEXT {
                 // We are on Windows 10 Anniversary Update (1607) or later.
-                if SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2)
-                    == FALSE
+                if !SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2)
+                    .as_bool()
                 {
                     // V2 only works with Windows 10 Creators Update (1703). Try using the older
                     // V1 if we can't set V2.
                     SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE);
                 }
             } else if let Some(SetProcessDpiAwareness) = *SET_PROCESS_DPI_AWARENESS {
                 // We are on Windows 8.1 or later.
-                SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
+                SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE).unwrap();
```

Comment:
Just pointing out, this is a semantic change

Reply:
@MarijnS95 recommended this. Should be fine I guess?

Reply:
`cargo check` should trigger here because of an unused `Result`. If unwrapping is not desired (ie. there are tangible fallible cases) this error should be bubbled up to the caller.